### PR TITLE
PIM-6607: redo upload zone

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

Here we redo the drop field for upload and import

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | NA
| Added Behats                      | NA
| Added integration tests           | NA
| Changelog updated                 | NA
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/job_instance/csv_product_import_show.yml [hunk 1]
```diff
@@ -61,6 +61,8 @@ extensions:
         module: pim/job/common/edit/upload
         parent: pim-job-instance-csv-product-import-show-upload-switcher-item
         position: 50
+        consig:
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/job_instance/csv_product_import_show.yml]: Is it a tpyo ?

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/export/common/edit/upload.html [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
-                <div class="AknMediaField-thumb file"><i class="AknMediaField-icon icon icon-file"></i></div>
+                <div class="AknMediaField-thumb file">
+                    <img src="/bundles/pimui/images/illustration-import.svg" alt="upload icon" class="AknMediaField-uploadIcon"/>
+                </div>
                 <div class="AknMediaField-info info">
                     <div class="filename" title="<%- file.name %>"><%- file.name %></div>
                     <div class="AknButtonList AknButtonList--centered actions">
-                        <span class="AknButtonList-item AknIconButton AknIconButton--grey clear-field"><i class="icon icon-trash"></i></span>
+                        <span class="AknButtonList-item clear-field">
+                            <img src="/bundles/pimui/images/icon-delete-slategrey.svg" class="AknMediaField-remove"/>
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/export/common/edit/upload.html]: We never do that, any reason here ?

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/field/media.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
     <% if (!value || !value.data || value.data.filePath === null) { %>
         <input class="AknMediaField-fileUploaderInput" id="<%- fieldId %>" type="file" data-locale="<%- locale %>" data-scope="<%- scope %>" <%- editMode === 'view' ? 'disabled' : '' %>/>
         <div class="AknMediaField-emptyContainer">
-            <img src="/bundles/pimui/images/upload.png" alt="upload icon" class="AknMediaField-uploadIcon"/>
+            <img src="/bundles/pimui/images/illustration-import.svg" alt="upload icon" class="AknMediaField-uploadIcon"/>
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/field/media.html]: idem

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/field/media.html [hunk 4]
```diff
@@ -13,7 +13,9 @@
             <% if ('pim_catalog_image' === attribute.type && null != mediaThumbnailUrl) { %>
                 <div class="AknMediaField-thumb file"><img src="<%- mediaThumbnailUrl %>" class="AknMediaField-image"/></div>
             <% } else { %>
-                <div class="AknMediaField-thumb file"><i class="AknMediaField-icon icon icon-file"></i></div>
+                <div class="AknMediaField-thumb file">
+                    <img src="/bundles/pimui/images/illustration-import.svg" alt="upload icon" class="AknMediaField-uploadIcon"/>
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/product/field/media.html]: idem

File: src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/job_instance/csv_product_import_show.yml (hunk 1 above)
Comment (juliensnz) [reply src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/job_instance/csv_product_import_show.yml]: noooooon

**Pull request merged**